.gitignore

```diff
@@ -5,4 +5,4 @@ htmlcov/
 .DS_Store
 dist/
 .ipynb_checkpoints
-
+.idea
```

Comment:
you may need to delete the .idea and inspectionProfiles files from the version of your repo on github, but then the .gitignore should handle them in the future!

Reply:
Okay, I think I just deleted those files.